### PR TITLE
Updated billing for harvard

### PR DESCRIPTION
**Pull request by Milstein**

### CODE REVIEW

File: docs/get-started/cost-billing/billing-process-for-harvard.md [hunk 1]
```diff
@@ -1,23 +1,34 @@
...
 change as needed per project. Please be cognizant that only a single billing code
 is allowed per billing asset. Therefore, if you have multiple projects with different
 funds, if you are able, please create a separate project for each fund. Otherwise,
 you will need to take care of this with internal journals inside of your department
-or lab. Each monthly billing cycle, the HUIT billing team will review and update
-the cost per asset. Subsequently, they will email billing cost information for
-each asset to the corresponding billing email addresses of all customers.
+or lab. During each monthly billing cycle, the NERC team will upload the billing
+Comma-separated values (CSV) files to the HUIT Central Billing system accessible
+AWS Object Storage (S3) bucket. The HUIT Central Billing system ingests billing
```
Comment (joachimweyl) [inline docs/get-started/cost-billing/billing-process-for-harvard.md:16]: Why are we using S3 when they already are using the sharedrive?

Comment (Milstein) [reply docs/get-started/cost-billing/billing-process-for-harvard.md:16]: HUIT Billing does not support the Shared Drive - they requires us to put the monthly CSVs for Harvard only to S3 bucket confirmed their parser to parse every day!

Comment (joachimweyl) [reply docs/get-started/cost-billing/billing-process-for-harvard.md:16]: MGHPCC & BUIT Billing have been using the sharedrive for invoicing for a long time. They are currently using it.

Comment (joachimweyl) [reply docs/get-started/cost-billing/billing-process-for-harvard.md:16]: Thank you for the in-person explanation. Now I understand that John G thought HUIT Billing was using the sharedrive and in fact, they were not.

**Review by joachimweyl — APPROVED** [review]

**Pull request merged**